### ISSUE TITLE
Is python 2.7 no longer available to download using code in ldsc tutorial?

### ISSUE THREAD

**Issue by laleoarrow** [bulik/ldsc#432]
Hi, all! Im running code on a macbook with m3max chip and Im experiencing difficulties "install" ldsc. Any help would be greatly appreciated!

I tried to install this conda env which is required in the environment.yml, but it prompt：
```
PackagesNotFoundError: The following packages are not available from current channels:

  - python=2.7*
  - pybedtools=0.7*
  - bitarray=0.8*

Current channels:

  - https://conda.anaconda.org/bioconda/osx-arm64
  - https://repo.anaconda.com/pkgs/main/osx-arm64
  - https://repo.anaconda.com/pkgs/r/osx-arm64
```

I have tried to download python2.7 separately, but unfortunately it can not work either. It seems that conda can not locat python2.7 to download. Or is it because I used wrong channels? Please advise.

**Comment by aksarkar** [comment]
@leoarrow1 No, python 2.7 is no longer available https://conda-forge.org/news/2020/03/18/python-27-and-vs2008-deprecation/

**Comment by laleoarrow** [comment]
If so, how should the required environment for ldsc be installed then? Is there perhaps a recommend way (for mac os)?

**Comment by aksarkar** [comment]
@leoarrow1 This repo is not maintained. Your best bet is to transform the source code using `2to3` and run in Python 3.

**Comment by laleoarrow** [comment]
> @leoarrow1 This repo is not maintained. Your best bet is to transform the source code using `2to3` and run in Python 3.

Thank you for clarifying this. Not sure why, but the standard tutorial works like a charm on linux server.

**Comment by charlesbreeze** [comment]
Update: NCI have released an updated Python 3 LDSC at https://github.com/cbiit/ldsc
